databricks/koalas/series.py

```diff
@@ -4258,6 +4258,60 @@ def xs(self, key, level=None):
 
         return _col(DataFrame(internal))
 
+    def pct_change(self, periods=1):
+        """
+        Percentage change between the current and a prior element.
+
+        .. note:: the current implementation of this API uses Spark's Window without
+            specifying partition specification. This leads to move all data into
+            single partition in single machine and could cause serious
+            performance degradation. Avoid this method against very large dataset.
+
+        Parameters
+        ----------
+        periods : int, default 1
+            Periods to shift for forming percent change.
+
+        Returns
+        -------
+        Series
+
+        Examples
+        --------
+
+        >>> kser = ks.Series([90, 91, 85])
+        >>> kser
+        0    90
+        1    91
+        2    85
+        Name: 0, dtype: int64
+
+        >>> kser.pct_change()
+        0         NaN
+        1    0.011111
+        2   -0.065934
+        Name: 0, dtype: float64
+
+        >>> kser.pct_change(periods=2)
+        0         NaN
+        1         NaN
+        2   -0.055556
+        Name: 0, dtype: float64
+        """
+        sdf = self._internal._sdf
+        window = Window.orderBy(self._internal.index_columns).rowsBetween(-periods, -periods)
+
+        for column_name in self._internal.data_columns:
+            prev_row = F.lag(F.col(column_name), periods).over(window)
+            sdf = sdf.withColumn(column_name, (F.col(column_name) - prev_row) / prev_row)
+
+        internal = _InternalFrame(
+            sdf=sdf,
+            index_map=self._internal.index_map,
+            column_index=self._internal.column_index)
+
+        return Series(internal)
```

Comment:
This loses the anchor. We should use `_col(DataFrame(internal))`.

Reply:
@ueshin ,thanks for the comment! 😸 

i have a question since i'm not pretty sure about something,

if we using `copy()` when make `internal` rather than `_InternalFrame`, then can use `Series(internal)`directly??

Reply:
Nope, we should create `DataFrame` first whenever we modify `sdf`.

Reply:
@ueshin 
ah, the matter was a `sdf`!
thanks very much for the explanation. now i totally got it. 👍

Reply:
Sorry, I mean, `Series(internal)` is not a proper constructor. We need to specify `anchor`.
`_col(DataFrame(internal))` is for hiding the construction of the `Series`.

Reply:
@ueshin 
ah, thanks for replying :) 
then how about `Series(internal, anchor=self)` ??

Reply:
`self` is not `DataFrame`.

Reply:
@ueshin thanks!! 😄